README.md

```diff
@@ -129,7 +130,26 @@ Touchégg is available from the main repository. To use it, you have to enable i
 $ sudo xbps-install touchegg
 $ sudo ln -s /etc/sv/touchegg /var/service
 ```
+## Alpine Linux
 
+Uncomment the url for the testing repository in /etc/apk/repositories, then install:
+
+```bash
+$ sudo apk update
+$ sudo apk add touchegg
+```
+
+The touchégg package includes an openrc init script for starting the touchégg daemon at boot. To enable:
+
+```bash
+$ sudo rc-update add touchegg`
+```
+The init script can also be used to manually start and stop the touchégg daemon as required:
```

Comment:
```suggestion
The init script can also be used to manually start and stop the Touchégg daemon as required:
```

Reply:
Corrected.

Reply:
Thank you for the review and sorry for the mistakes. I am the maintainer for the Alpine package and just had the package build merged this past weekend, so it is a new addition to the repos. It will spend some time in the testing repo and then eventually move to community, but that must be requested via another merge request. The install instructions would need to be updated at that time. I'm not sure what Alpine's criteria is for approving the move from testing -> community.

The package can be seen here:
https://pkgs.alpinelinux.org/packages?name=touchegg&branch=edge